synapse/storage/__init__.py

```diff
@@ -266,6 +267,30 @@ def _count_users(txn):
 
         return self.runInteraction("count_users", _count_users)
 
+    def count_monthly_users(self):
+        """
+        Counts the number of users who used this homeserver in the last 30 days
+        This method should be refactored with count_daily_users - the only
+        reason not to is waiting on definition of mau
+        returns:
+            defered: resolves to int
```

Comment:
FWIW, we generally start the docstring on the sameline of the quotes, so something like:

```python
"""Counts the number of users who used this homeserver in the last 30 days.

This method should be refactored with count_daily_users - the only
reason not to is waiting on definition of MAU.

Returns:
    Deferred[int]
```

And that's also how you annotate return types (such that IDEs correctly interpret it)